cves/CVE-2016-3092.yml

```diff
@@ -206,6 +211,11 @@ mistakes:
     Look at the CWE entry for this vulnerability and examine the mitigations
     they have written there. Are they doing those? Does the fix look proper?
     Use those questions to inspire your answer. Don't feel obligated to answer
-    every one. Write a thoughtful entry here that those ing the software
+    every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    I think this was a coding mistake because, when getting string input, it is good
+    practice to check the length of the string to make sure that it won't cause any
+    problems. I think the fix looks proper because it sets the buffer size to the
+    boundary length times two if the boundary string is too large. This helps to prevent
```

Comment:
Isn't a check of the input for size and increasing buffer size overkill?

Reply:
I will address this when making my edits.

Reply:
Looking back on it, I think that the error can be caused by either a boundary string that is too long or one that is just smaller than the buffer size. The buffer size increase only seems to happen if the boundary string is just below the buffer size, so I don't think the fix is too overkill.